receiver/snowflakereceiver/documentation.md

```diff
@@ -0,0 +1,84 @@
+| **snowflake.rows_produced.sum** | rows produced | rows | Gauge(Int) | <ul> <li>schema.name</li> <li>execution.status</li> <li>error.message</li> <li>query.type</li> <li>warehouse.name</li> <li>database.name</li> <li>warehouse.size</li> </ul> |
+| **snowflake.rows_unloaded.sum** | rows unloaded | rows | Gauge(Int) | <ul> <li>schema.name</li> <li>execution.status</li> <li>error.message</li> <li>query.type</li> <li>warehouse.name</li> <li>database.name</li> <li>warehouse.size</li> </ul> |
+| **snowflake.rows_updated.sum** | rows updated | rows | Gauge(Int) | <ul> <li>schema.name</li> <li>execution.status</li> <li>error.message</li> <li>query.type</li> <li>warehouse.name</li> <li>database.name</li> <li>warehouse.size</li> </ul> |
+| **snowflake.session_id.count** | distinct session ids | 1 | Gauge(Int) | <ul> <li>user.name</li> </ul> |
+| **snowflake.storage.failsafe_bytes.total** | total failsafe bytes in snowflake | bytes | Gauge(Int) | <ul> </ul> |
+| **snowflake.storage.stage_bytes.total** | total stage bytes in snowflake | bytes | Gauge(Int) | <ul> </ul> |
+| **snowflake.storage.storage_bytes.total** | total storage bytes in snowflake | bytes | Gauge(Int) | <ul> </ul> |
+| **snowflake.total_elapsed_time.avg** | average elapsed time | s | Gauge(Double) | <ul> <li>schema.name</li> <li>execution.status</li> <li>error.message</li> <li>query.type</li> <li>warehouse.name</li> <li>database.name</li> <li>warehouse.size</li> </ul> |
+| **snowflake.total_elapsed_time.sum** | total elapsed time | s | Gauge(Int) | <ul> <li>schema.name</li> <li>execution.status</li> <li>error.message</li> <li>query.type</li> <li>warehouse.name</li> <li>database.name</li> <li>warehouse.size</li> </ul> |
+
+**Highlighted metrics** are emitted by default. Other metrics are optional and not emitted by default.
+Any metric can be enabled or disabled with the following scraper configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: <true|false>
+```
+
+## Resource attributes
+
+| Name | Description | Type |
+| ---- | ----------- | ---- |
+| snowflake.account.name | The name of the snowflake account being monitored | String |
+| snowflake.username | The name of the snowflake user account being used | Empty |
+| snowflake.warehouse.name | The name of the warehouse being used | String |
+
+## Metric attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| database.name | name of database (default is snowflake) |  |
+| error.message | Error message if present |  |
+| execution.status | execution status |  |
+| is.success | Login status |  |
+| pipe.name | name of snowpipe |  |
+| query.type | type of query performed |  |
+| reported.client.type | Client type used for attempt |  |
+| schema.name | name of schema |  |
+| service.type | Service type |  |
+| user.name | username in query |  |
+| warehouse.name | name of Warehouse in query |  |
```

Comment:
Why do we have warehouse name both in metric attributes and resource attributes?